mantidproject/mantid
# Migrate mantid basics to rst

moved mantid basics to rst pages

fixes #13114

NOTE: Have not moved links on the wiki yet until this is approved, and we have proven we can create a pdf from it.
### To test
1. check that the build passes without warnings
2. take a look at some of the pages.
